<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidgetClass</class>
 <widget class="QMainWindow" name="MainWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Btn_Last">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_StartAndStop">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Next">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="PlayerSlider" name="slider_video">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>390</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_voiceIcon">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_curTime">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_speed1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>× 0.5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_speed2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>× 1.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_speed3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>× 1.2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_speed4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>× 1.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_maxTime">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>5</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_speed5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>×2.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Min">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Max">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Close">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QVideoWidget" name="widget_video" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>650</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_loadFile">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="slider_voice">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>240</width>
      <height>430</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayerSlider</class>
   <extends>QSlider</extends>
   <header>playerslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWidget.qrc"/>
 </resources>
 <connections/>
</ui>
